site/styles/_variables.scss

```diff
@@ -47,9 +47,9 @@
   --page-padding-top: 120px;
   --page-padding-top-md: 120px;
   --page-padding-top-lg: 180px;
-  --page-detail-padding-top: 120px;
-  --page-detail-padding-top-md: 120px;
-  --page-detail-padding-top-lg: 180px;
+  --page-detail-padding-top: 245px;
+  --page-detail-padding-top-md: 290px;
+  --page-detail-padding-top-lg: 440px;
```

Comment:
Are these padding values based on some sort of calculation?  Or just more random numbers?

Reply:
More random - they would be the result of matching to the designs, but then rounding to the nearest 5px